docs/versioned_docs/version-v2.4/03-Guidance/use-with-dotnet-and-functions.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Using Arcus & Serilog in .NET Core and/or Azure Functions"
+layout: default
+---
+
+# Using Arcus & Serilog in .NET Core and/or Azure Functions
```

Comment:
Is it still relevant to say 'in .NET Core'  , now that we also support .NET 6 ?

Reply:
Yeah, that's indeed something that's different now. Hmm, if we change the title, we should make sure that we add an redirect bc it's a H1 title.

Reply:
`Using Arcus & Serilog in Azure Functions`?

Reply:
Updated this.